Correct JSON-RPC docs for getSlot (#19355)
solana-labs/solana: docs/src/developing/clients/jsonrpc-api.md

### getSlot

> Returns the slot associated with the given commitment level
> 
> If the commitment level is not provided, then this method returns the highest
> finalized slot.

Reviewer: ```suggestion
Returns the slot that has reached the [given or default commitment level](jsonrpc-api.md#configuring-state-commitment)
```
